mirror/images.txt

```diff
@@ -8,25 +8,11 @@
 
 # ceph-csi devel
 docker.io/rook/ceph:v1.12.1      rook/ceph:v1.12.1
```

Comment:
will update Rook version based on https://github.com/ceph/ceph-csi/pull/4736 testing

Reply:
Need this PR to get in to test latest Rook and ceph version